workflow/infrastructure-monitoring/container-monitoring.mdx

```diff
@@ -1,73 +1,131 @@
+        </Accordion>
 
-![](/images/infra-monitoring/c5.png)
+        <Accordion title="Jobs">
+            Jobs are controller objects that represent finite and short-lived tasks or batch processes that run within pods.
+        </Accordion>
+
+        <Accordion title="CronJobs">
+            CronJobs are controller objects that can be scheduled to automate task execution. CronJobs are Jobs that can be run on a periodic schedule.
+        </Accordion>
+
+        <Accordion title="DaemonSet">
+            DaemonSets are controllers that ensure a specific set of pods is running across your cluster. DaemonSets are used to run exactly one instance of a pod on each node in the cluster.
+        </Accordion>
+
+        <Accordion title="StatefulSet">
+            StatefulSets are used for managing stateful applications that are comprised of unique network identifiers, persistent storage, and specific ordering requirements for their deployment and scaling. 
+        </Accordion>
+
+        </AccordionGroup>
+
+</Accordion>
+
+# Usage Graph
+
+The Usage graph displays all of your team's Infrastructure usage data. Gain immediate insight into your stack's performance.
+
+### Change Usage Type
+
+Toggle between your Infrastructure CPU and Memory usage over time with the Usage Graph.
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaDRkZHlldGN3bWJ4b281emh3Ym52ZHFpaGJ4anF2OWtzZTMyMW9vbiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/CMxUd2XqN2sQKFTXEW/giphy.gif" width="750" height="750"/>
+
+### Select Date Range
+
+Select your own date range from the calendar or choose from our default options.
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExMWpuamp5amFsM3dtazFobXVibnhpb2xmc3NqOWN6d3djczM4anQydCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/9ZT90BmYhPrqB3fiWt/giphy.gif" width="750" height="750"/>
+
+# Data Table
+
+The List Tab displays all of your specific Infrastructure data in a single table. You can manage your list view by adding and removing specific metrics.
+
+### Add & Remove Data
+Select specific Infrastructure data you would like to see in your data table by toggling on and off each metric.
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExNzhsdm1ycHB5b244a3B1ZTJnaDNjOGlmZzFteDFob2FrZjAzdG1nZiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/PtGflEPrP0lFAGdV4i/giphy.gif" width="750" height="750"/>
+
+### Filter & Search
+Access specific Hosts, Host IDs, Host Names, K8s Nodes, OS Types, and other key attributes with the Filters bar. Quickly index across all of your infrastructure data in one search. 
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZXJmNDM5d2JmYzd4aTJ2OHg3ZHd3dms2bTJwZ3R1Z3dzcXl0anNkeiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/rEJAUGmdTiJZqvOEBT/giphy.gif" width="750" height="750"/>
+
+### Live & History View
+Toggle between a static data table with the History View or track your Infrastructure data in real-time with Live View.
```

Comment:
Worth a note as a second sentence that whenever a time range is selected the application automatically goes into history view. I see user slip up on this constantly.